docs/book/known_issues.md

```diff
@@ -75,3 +80,23 @@ Issue 9<a id="issue_9"></a>: CnsRegisterVolume API does not validate if the volu
 
 - Impact: One of the PVC using the CNS volume will be usable at any point in time. Usage of any other PVC will lead to attach failures.
 - Workaround: None
+
+Issue 10<a id="issue_10"></a>: Volume expansion might fail when it is called with pod creation simultaneously.
+
+- Impact: Users can resize the PVC and create a pod using that PVC simultaneously. In this case, pod creation might be completed first using the PVC with original size. Volume expansion will fail because online resize is not supported in vSphere 7.0 Update1.
+- Workaround: None
+
+Issue 11<a id="issue_11"></a>: Supervisor devops can manually expand the PVC in Supervisor namespace in vSphere 7.0 Update1.
+
+- Impact: Supervisor devops can manually expand the PVC in Supervisor namespace, but the file system will not be expanded. It is not a supported use case and a current limitation now.
+- Workaround: Create a static PVC in Tanzu Kubernetes Grid Service Cluster using the SVC PVC, and expand the TKGS PVC to a size equal to or greater than the previously requested size and then attached to a Pod for the underlying filesystem to resize.
+
+Issue 12<a id="issue_12"></a>: The Supervisor Cluster PVC expanded size is lost and cannot be reclaimed in vSphere 7.0 Update1.
+
+- Impact: Users can create a PVC and expand the volume in a Tanzu Kubernetes Grid Service Cluster. Before the TKGS PVC is attached to a pod, the TKGS is destroyed. The Supervisor Cluster PVC can be reused in a different TKGS with FilesystemResizePending condition, but filesystem will not be expanded.
```

Comment:
Rephrase: `Supervisor Cluster PVC can be reused in a different TKGS with FilesystemResizePending condition, but filesystem will not be expanded` -> `Supervisor Cluster PVC with its FilesystemResizePending condition can be reused in a different TKGS but filesystem will not be expanded`

Reply:
Done.